src/Microsoft.SqlTools.ServiceLayer/QueryExecution/ResultSet.cs

```diff
@@ -589,6 +591,28 @@ private void SingleColumnXmlJsonResultSet()
             }
         }
 
+        /// <summary>
+        /// Check columns for json type and set isJson if needed
+        /// </summary>
+        private void CheckForIsJson()
+        {
+            if (Columns?.Length > 0 && RowCount != 0)
+            {
+                Regex regex = new Regex(@"({.*?})");
```

Comment:
will this regex have a lot of false positives?  (i.e., non JSON content tagged as JSON?).  Overall I'm fine with this and we can tune the regex is get customer complaints, though if there is a simple check to the make the regex a little more restrictive that may be worth doing now...for example, is there at least on field separated be a ':' or something like that.